src/Files.App/Views/Layouts/GridLayoutPage.xaml.cs

```diff
@@ -430,15 +430,17 @@ private async void FileList_ItemTapped(object sender, TappedRoutedEventArgs e)
 						if (FolderSettings.LayoutMode == FolderLayoutModes.GridView)
 						{
 							Popup popup = gridViewItem.FindDescendant("EditPopup") as Popup;
-							var textBox = popup.Child as TextBox;
+							var textBox = popup?.Child as TextBox;
 
-							await CommitRenameAsync(textBox);
+							if (textBox is not null)
+								await CommitRenameAsync(textBox);
```

Comment:
I don't think this makes sense because `GridLayoutPage.EndRename` doesn't appear to throw a NullReferenceException even if `textBox` is null.

Reply:
What do you think is causing the exception?

Reply:
Maybe we need to consider the possibility of `popup` and `textBlock` being null.
https://github.com/files-community/Files/blob/fa2caab676b86447dbec45fc0f85b4a2106466b0/src/Files.App/Views/Layouts/GridLayoutPage.xaml.cs#L252-L264